packages/app-mobile/components/screens/ConfigScreen/JoplinCloudConfig.tsx

```diff
@@ -0,0 +1,75 @@
+import { _ } from '@joplin/lib/locale';
+import * as React from 'react';
+import { View, Text, Linking } from 'react-native';
+import Clipboard from '@react-native-clipboard/clipboard';
+import SettingsButton from './SettingsButton';
+import { accountTypeToString } from '@joplin/lib/utils/joplinCloud';
+import { LinkButton } from '../../buttons';
+import { ConfigScreenStyles } from './configScreenStyles';
+
+type JoplinCloudConfigProps = {
+	styles: ConfigScreenStyles;
+	accountType: string;
+	inboxEmail: string;
+	website: string;
+	userEmail: string;
+};
+
+export const emailToNoteLabel = () => _('Email to note');
+export const emailToNoteDescription = () => _('Any email sent to this address will be converted into a note and added to your collection. The note will be saved into the Inbox notebook');
+export const accountInformationLabel = () => _('Account information');
+export const accountTypeLabel = () => _('Account type');
+export const accountEmailLabel = () => _('Email');
+
+const JoplinCloudConfig = (props: JoplinCloudConfigProps) => {
+
+	const isEmailToNoteAvailableInAccount = props.accountType !== '1';
+	const inboxEmailValue = props.inboxEmail ?? '-';
+
+	const goToJoplinCloudProfile = async () => {
+		await Linking.openURL(`${props.website}/users/me`);
+	};
+
+	return (
+		<View key="joplinCloud">
+			<View style={props.styles.styleSheet.settingContainerNoBottomBorder}>
+				<Text style={props.styles.styleSheet.settingTextEmphasis}>{accountInformationLabel()}</Text>
+			</View>
+			<View style={props.styles.styleSheet.settingContainerNoBottomBorder}>
+				<Text style={props.styles.styleSheet.settingText}>{accountTypeLabel()}</Text>
+				<Text style={props.styles.styleSheet.settingTextEmphasis}>{accountTypeToString(parseInt(props.accountType, 10))}</Text>
```

Comment:
Be careful here! Currently, `accountTypeToString` throws when the account type is unknown. If we eventually add a new account type, this might cause the app to crash if users open the JoplinCloud config screen on an old client.

Consider including a check that `accountType` actually is a valid account type, and displaying `'Unknown'` if not.